mesonbuild/backend/ninjabackend.py

```diff
@@ -2282,7 +2282,7 @@ def get_user_option_args(self):
     def generate_utils(self, outfile):
         cmd = [sys.executable, self.environment.get_build_command(),
                '--internal', 'scanbuild', self.environment.source_dir, self.environment.build_dir,
-               sys.executable, self.environment.get_build_command()] + self.get_user_option_args()
+               sys.executable, self.environment.get_build_command()] + sorted(self.get_user_option_args())
```

Comment:
I couldn't work out how to remove this one. The commandline arguments don't appear to have a stable ordering at any point. We could move the sorted() call somewhere earlier ...